web/cobrands/sass/_base.scss

```diff
@@ -985,12 +986,7 @@ input.final-submit {
 }
 
 .problem-header {
-  margin-bottom:1em;
-}
-.problem-header .update-img {
-    float: $right;
-    margin-#{$left}: 0.5em;
-    margin-bottom: 0.5em;
+  margin-bottom: 1em;
```

Comment:
As you asked in IRC but have gone offline, it looks like this margin bottom is to provide some extra separation in mobile view between the main problem description and the key tools links bar.